packages/core/libs/api/NotionUser.ts

```diff
@@ -233,7 +233,6 @@ class NotionUser extends Data<INotionUser> {
       } else if (page.type === "collection_view_page"){
         await this.initializeCacheForSpecificData(id, 'block');
```

Comment:
this  is the only way the code works for me. i'm honestly not sure why i need to call this, i just know the cache didn't have the collection

Reply:
Yes exactly, since the cache wasn't initialized the collection would not exist in the cache and it would be undefined, but it should be called only once, at the beginning. I can see from your forked repo that its called twice in line 234 and then again in line 239.

Reply:
oh! you want me to remove the second call? ok

Reply:
Yes that would also mean that the test related to it has to be updated, the mock for `initializeCacheForSpecificData` should be called 2 times rather than 3.